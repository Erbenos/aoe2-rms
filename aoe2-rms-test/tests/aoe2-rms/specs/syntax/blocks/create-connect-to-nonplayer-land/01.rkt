#lang aoe2-rms
; Connect players to a central desert, but not directly to each other

<LAND-GENERATION>
(create-player-lands
 (terrain-type 'DIRT2)
 (number-of-tiles 100))

(create-land
 (terrain-type 'DESERT)
 (number-of-tiles 500)
 (land-position 50 50))

<CONNECTION-GENERATION>
(create-connect-to-nonplayer-land
 (replace-terrain 'GRASS 'ROAD2))
