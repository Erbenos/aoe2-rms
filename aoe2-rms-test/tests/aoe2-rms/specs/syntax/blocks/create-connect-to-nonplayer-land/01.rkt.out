<LAND_GENERATION>
create_player_lands {
  terrain_type DIRT2
  number_of_tiles 100
}
create_land {
  terrain_type DESERT
  number_of_tiles 500
  land_position 50 50
}
<CONNECTION_GENERATION>
create_connect_to_nonplayer_land {
  replace_terrain GRASS ROAD2
}
